src/voice.ts

```diff
@@ -159,7 +159,7 @@ export class Voice extends Element {
       let boundingBox = undefined;
       for (let i = 0; i < this.tickables.length; ++i) {
         const tickable = this.tickables[i];
-        tickable.setStave(stave);
+        if (!tickable.getStave()) tickable.setStave(stave);
```

Comment:
Just curious about this line. Did it fix a bug? Is there something wrong with blindly calling .setStave() every time? :-)

Reply:
Ron thanks for the review! The tickabless of a voice can be located on different staves. The voice has a defect stave, The tickables should only be set to the default if they have no previously associated stave.

Reply:
The aim of this PR is to allow Voices have Tickables in several Staves and to make sure that System also allows it.